### DISCUSSION TITLE
Window 10 32bit installation on Mac using OVMF

### DISCUSSION THREAD

**Discussion by Jin050812** [Homebrew/discussions#3753] [Everyday usage]
I am trying to download Window 10 on a macbook air but I am having difficulties on terminal.. can anyone tell me how to install Winodw on mac without using bootcamp? 
<img width="993" alt="Screen Shot 2022-10-07 at 12 41 58 PM" src="https://user-images.githubusercontent.com/115205880/194469218-294b5298-f868-4a3a-a32c-162ca30e2e0f.png">

**Comment by SMillerDev** [comment]
That seems more like a question for stackoverflow. Homebrew can't be used to install windows.
